CMSIS/Core_A/Source/irq_ctrl_gic.c

```diff
@@ -65,6 +71,14 @@ __WEAK int32_t IRQ_SetHandler (IRQn_ID_t irqn, IRQHandler_t handler) {
   return (status);
 }
 
+/// The Interrupt Handler.
+__WEAK void IRQ_Handler (void) {
+  IRQn_Type irqn = GIC_AcknowledgePending ();
+  if (irqn < (IRQn_Type)IRQ_GIC_LINE_COUNT)
+    IrqTable[irqn]();
```

Comment:
Please add braces `{ }` around the block to prevent merge anomalies.

```suggestion
  if (irqn < (IRQn_Type)IRQ_GIC_LINE_COUNT) {
    IrqTable[irqn]();
  }
```

Reply:
I’ve added braces around the statement.